src/css/styles.scss

```diff
@@ -1,85 +1,83 @@
+@import 'variables';
+@import 'mixins';
+// @import 'reset'
+
 *,
 *:after,
 *:before {
-box-sizing: border-box;
-margin: 0;
-padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
 body {
-  background-color: #1F1F1F;
-  font-family: 'Oswald', sans-serif;
+  background-color: $background-color;
+  font-family: $primary-font-body;
 }
-
 /* HEADER */
 header {
   text-align: center;
   height: 9vw;
+  h1 {
+    color: $white-color;
+    display: inline;
+    font-size: 5.4vw;
+    letter-spacing: 0.2em;
+  }
+  .profile-button {
+    background: none;
+    font-size: 5.6vw;
+    border: none;
+    display: inline;
+    float: right;
+    height: 5.6vw;
+    margin-right: 0.5vw;
+    margin-top: 1.6vw;
+    width: 5.6vw;
+    color: $white-color;
+  }
+  #user-info-dropdown {
```

Comment:
We will have to go back through these later and make sure we change any id's that they used to camelCase!